docs/vcs.rst

```diff
@@ -390,48 +379,47 @@ while :ref:`vcs-gitlab` creates merge request.
 
 .. _gitlab-push:
 
-Pushing changes to GitLab as merge request
+Pushing changes to GitLab as merge requests
 ++++++++++++++++++++++++++++++++++++++++++
 
-If you are translating a project that is hosted on GitLab and don't want to
-push translations to the repository, you can have them sent as a merge request.
+If not wanting to push translations to a GitLab repository, they can be sent as either
+one or many merge requests instead.
 
-You need to configure the `lab`_ command line tool and set
-:setting:`GITLAB_USERNAME` for this to work.
+Configure the `lab`_ command line tool and set :setting:`GITLAB_USERNAME` for this to work.
 
 .. seealso::
 
    :setting:`GITLAB_USERNAME`, :ref:`lab-setup` for configuration instructions
 
 .. _lab-setup:
 
-Setting up lab
+Setting up Lab
 ++++++++++++++
 
-:ref:`gitlab-push` requires a configured `lab`_ installation on your
-server. Follow the installation instructions at
-https://github.com/zaquestion/lab#installation and perform and run it without
-any arguments to finish configuration, for example:
+:ref:`gitlab-push` requires a configured `lab`_ installation on your server.
+Follow the installation instructions at
+https://github.com/zaquestion/lab#installation and run it without any arguments to
+finish the configuration, for example:
 
 .. code-block:: sh
 
     # DATA_DIR is set in Weblate settings.py, set it accordingy.
-    # Is is /app/data in Docker
+    # It is /app/data in Docker
     $ HOME=${DATA_DIR}/home lab
-    Enter GitLab host (default: https://gitlab.com):
+    Enter the GitLab host (default: https://gitlab.com):
     Create a token here: https://gitlab.com/profile/personal_access_tokens
-    Enter default GitLab token (scope: api):
-    Config saved to ~/.config/lab.hcl
+    Enter the default GitLab token (scope: api):
+    (The config is saved to ~/.config/lab.hcl)
```

Comment:
This is output of the lab program, in case you want to improve it, please submit it to them :-).